package/MDAnalysis/coordinates/PDB.py

```diff
@@ -1179,10 +1182,36 @@ def _write_timestep(self, ts, multiframe=False):
             vals['tempFactor'] = temp if temp is not None else 0.0
             vals['segID'] = segid[:4]
             vals['element'] = guess_atom_element(atom.name.strip())[:2]
-
             # .. _ATOM: http://www.wwpdb.org/documentation/format32/sect9.html
+            
+            if i != 0 and vals['chainID'] != prev_atom['chainID'] \
+                and vals['chainID'] != " " and prev_atom['chainID'] != " ":
```

Comment:
Never have the continuation of a `if`, `for`, `while`... align with the block it introduces. Here you should add an extra indentation to avoid confusion.